src/dsp_tools/resources/4124-template-repo/template.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version='1.0' encoding='utf-8'?>
+
+<knora xmlns="https://dasch.swiss/schema"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="https://dasch.swiss/schema https://raw.githubusercontent.com/dasch-swiss/dsp-tools/main/src/dsp_tools/schemas/data.xsd"
+       shortcode="4124"
+       default-ontology="testonto">
+
+    <permissions id="res-default">
+        <allow group="UnknownUser">RV</allow>
+        <allow group="KnownUser">V</allow>
+        <allow group="Creator">M</allow>
+        <allow group="ProjectAdmin">CR</allow>
+    </permissions>
+    <permissions id="prop-default">
+        <allow group="UnknownUser">V</allow>
+        <allow group="KnownUser">V</allow>
+        <allow group="Creator">D</allow>
+        <allow group="ProjectAdmin">CR</allow>
+    </permissions>
+
+    <resource label="The only resource" restype=":minimalResource" id="res_0" permissions="res-default">
+        <text-prop name=":hasText">
+            <text encoding="utf8" permissions="prop-default">Hello!</text>
+        </text-prop>
```

Comment:
here, maybe you could have one for each possible `<xxx-prop>` type, so it's quicker to copy-paste around?

Reply:
And also in the ontology? My initial thought was to create two files that are compliant, and can be used as they are. If I add different properties, I'm inclined to add them in the onto as well...

Reply:
that relates to what you think about my other comment... that would require you to add things like `hasText`, `hasInt`, etc., which I'm not necessarily too fond of, as I would find myself most of the time just deleting them again.  
For me it doesn't have to be compliant, because if I want to upload something I don't have to modify first, I can now do `dsp-tools rosetta` :)